docs/source/examples/frameworks/gke_dws.md

```diff
@@ -0,0 +1 @@
+../../generated-examples/gke_dws.md
```

Comment:
We should also update this page: https://docs.skypilot.co/en/latest/reservations/reservations.html#gcp-dynamic-workload-scheduler-dws